tools/test-renode.mk

```diff
@@ -0,0 +1,139 @@
+TEST_UPDATE_VERSION?=2
+WOLFBOOT_VERSION?=0
+RENODE_UART=/tmp/wolfboot.uart
```

Comment:
Would it make sense to use `$TMP` isntead of `/tmp` or maybe even `mktemp` (may be less compatible)